models/demos/t3000/llama2_70b/tt/llama_model_optimized.py

```diff
@@ -235,7 +232,7 @@ def prepare_inputs(self, inp_ids, start_pos, valid_seq_len=None, mode="decode",
                 cos_gathered,
                 dtype=ttnn.bfloat16,
                 layout=ttnn.TILE_LAYOUT,
-                cache_file_name=cache_name(f"cos_gathered_prefill_{seq_len}"),
+                cache_file_name=cache_name(f"cos_gathered_prefill_{start_pos}_{start_pos+seq_len}"),
```

Comment:
This is to resolve a cache conflict that we would get otherwise with chunking. I don't think the cache should blow up, only that chunked cos/sin will be properly cached separately